Update "Choosing between Zulip Cloud and self-hosting" and "Logging in" help pages (#23341)
zulip/zulip: templates/zerver/help/logging-in.md

  page if your Zulip organization is hosted on [Zulip
  Cloud](https://zulip.com/plans/).

* You can ask your organization administrators for your Zulip URL.

{tab|logged-in}

* If using Zulip in the browser, your organization's Zulip log in URL is the first part
  of what you see in the URL bar (e.g., `<organization-name>.zulipchat.com` for
  [Zulip Cloud](https://zulip.com/plans/) organizations).

* In the Desktop app, select **Copy Zulip URL** from the **Zulip** menu to
  copy the URL of the currently active organization. You can also access the
  **Copy Zulip URL** option by right-clicking on an organization logo in the
  **organizations sidebar** on the left.

* In the Mobile app, tap your **profile picture** in the bottom right corner of
  the app, then tap **switch account** to see the URLs for all the organizations
  you are logged in to.

* On [Zulip Cloud](https://zulip.com/plans/) and other Zulip servers updated to
  [Zulip 6.0 or
  higher](https://zulip.readthedocs.io/en/latest/overview/changelog.html#zulip-6-x-series),
  click on the **gear** (<i class="fa fa-cog"></i>) icon in the upper right

>   corner of the web or desktop app. The organization URL is shown in the top

Reviewer: Perhaps we should tweak this a bit for consistency:

```suggestion
  corner of the web or desktop app. Your organization's log in URL is shown in the top
```

{end_tabs}

## Switch between organizations

{!switching-between-organizations.md!}

## Set or reset your password

> If you signed up using passwordless authentication and want to start logging in

Reviewer: This might convey that the passwordless authentication will stop working if they choose to set a password. Maybe we should say something like:

```suggestion
If you signed up using passwordless authentication and want to have the option to log in
```

> via email/password, you will need to reset your password. You can also reset a

Reviewer: Maybe saying "create" instead of "reset" would be a better way to phrase it given they've been using passwordless auth.
 
```suggestion
via email/password, you will need to create a password by following the instructions below. You can also reset a
```
